project/hackyourweather-2/.gitignore

```
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
```

so once you have a key you will use, you either have in inside a `.env.local` as react has ignored this already, or:
```suggestion
.env

```